host/4/bullseye/amd64/node/node14/node14-composite.template

```diff
@@ -26,9 +31,7 @@ ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
 RUN apt-get update && \
     apt-get install -y libc-dev
 
-COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
 COPY --from=runtime-image [ "/workers/node", "/azure-functions-host/workers/node" ]
```

Comment:
Should we pull the node worker ibefore we do setup_14.x / setup_16.x so that the worker layer is shared? Its probably not a huge saving, but it would also make it more consistent  with some of the other files that already have the worker at the top.

Reply:
Yeah I will do this now. thank you Paul

Reply:
Updated node to share worker portion.  Saved another .3mb in shared sizes. Thanks Paul
![image](https://user-images.githubusercontent.com/46761504/154363168-9ec1eb82-b97a-4dec-baa9-911f1658be02.png)